jsapp/js/account/usage/usageProjectBreakdown.tsx

```diff
@@ -1,158 +1,100 @@
-      .filter((projectResult) => {
-        const matchingAsset = assetsData.results.find(
-          (asset: ProjectViewAsset) => asset.uid === projectResult.uid
-        );
-
-        // Include the asset in the updatedData only if it is a survery
-        return matchingAsset?.asset_type === 'survey';
-      })
-      .map((projectResult) => {
-        const matchingAsset = assetsData.results.find(
-          (asset: ProjectViewAsset) => asset.uid === projectResult.uid
-        );
-
-        if (matchingAsset) {
+   useEffect(() => {
+    async function fetchData() {
+      try {
+        const data = await getAssetUsageForOrganization();
+        console.error(data);
+        const updatedResults = data.results.map((projectResult) => {
+          const assetParts = projectResult.asset.split('/');
+          const uid = assetParts[assetParts.length - 2];
           return {
             ...projectResult,
-            assets: [matchingAsset],
-            deployment_status: matchingAsset.deployment_status,
+            uid: uid,
           };
+        });
+
+        if (isComponentMounted) {
+          setProjectData({
+            ...data,
+            results: updatedResults,
+          });
         }
-        return projectResult;
-      });
-
-    setProjectData((prevData: Project) => {
-      return {
-        ...prevData,
-        results: updatedData as ProjectResult[],
-      };
-    });
-  }
-};
+      } catch (error) {
+        console.error('Error fetching data:', error);
+      }
+    }
 
-  // This code will be updated to only have one api call when the asset_usage api is updated
-  // Right now, you may need to refresh or re-navigate to the page multiple times as the Loading Spinner can be shown indefinitely
-  useEffect(() => {
-    const usageUrl = `${ROOT_URL}/api/v2/asset_usage`;
-
-    const fetchData = $.ajax({
-      dataType: 'json',
-      method: 'GET',
-      url: usageUrl,
-    })
-      .done(onFetchAssetsDone)
-      .then(() => {
-        const assetsUrl = `${ROOT_URL}/api/v2/assets`;
-
-        return $.ajax({
-          dataType: 'json',
-          method: 'GET',
-          url: assetsUrl,
-        })
-          .done((assetsData) =>
-            onFetchAssetsDataDone(assetsData, projectData.results)
-          )
-          .fail(onAnyFail);
-      })
-      .fail(onAnyFail);
+    fetchData();
 
     return () => {
       setIsComponentMounted(false);
     };
+  }, [isComponentMounted]);
 
-    function onAnyFail(error: any) {
-      console.error('Error fetching data:', error);
-    }
-  }, [projectData.results]);
-
-  if (projectData.results.length === 0 || projectData.results.some((result) => !result.assets)) {
+  if (projectData.results.length === 0) {
     return <LoadingSpinner/>;
   }
 
+const calculateRange = (): string => {
+  const totalProjects = parseInt(projectData.count);
+  const startRange = (currentPage - 1) * 8 + 1;
+  const endRange = Math.min(currentPage * 8, totalProjects);
```

Comment:
The page size of `8` should get turned into a constant, here and anywhere else it shows up in the front-end 👍 It'll make our lives easier if, say, someone asks us to show 50 results per page somewhere down the line 😅